utils/json/Cargo.toml

```diff
@@ -20,3 +20,9 @@ multihash = { workspace = true, default-features = false, features = [
 ] }
 num-bigint.workspace = true
 serde = { workspace = true, features = ["derive"] }
+
+[dev-dependencies]
+forest_json.workspace = true
+quickcheck            = { version = "1" }
+quickcheck_macros     = "1"
+serde_json            = "1.0"
```

Comment:
It would be great to have those versions also in the workspace and inherit from them.